index.js

```diff
@@ -9,7 +9,9 @@ function fileExists (filepath, options, done = function () {}) {
 
   fs.stat(fullPath(filepath, options), (err, stats) => {
     if (err) {
-      return done(err)
+      return err.code === "ENOENT" ?
+        done(null, false) :
```

Comment:
I know this feels a bit nit-picky, but can you change this to:

```js
return err.code === 'ENOENT'
  ? done(null, false)
  : done(err)
```

Also, why is it that we return `null` as the error if the file is missing?